Error when building

Seems like related to ffmpeg error, lot of my AUR is also affected.
```
[4/543] Building CXX object plugins/core/Standard/qAnimation/ex...QTFFmpegWrapper/CMakeFiles/QTFFMPEG_LIB.dir/QVideoEncoder.cpp.o
FAILED: plugins/core/Standard/qAnimation/extern/QTFFmpegWrapper/CMakeFiles/QTFFMPEG_LIB.dir/QVideoEncoder.cpp.o 
/usr/bin/c++ -DQT_CORE_LIB -DQT_GUI_LIB -DQT_NO_DEBUG -DQT_USE_QSTRINGBUILDER -D__STDC_CONSTANT_MACROS -I/home/toni/.cache/yay/cloudcompare-git/src/cloudcompare/plugins/core/Standard/qAnimation/extern/QTFFmpegWrapper -isystem /usr/include/qt -isystem /usr/include/qt/QtGui -isystem /usr/include/qt/QtCore -isystem /usr/lib/qt/mkspecs/linux-g++ -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fexceptions         -Wp,-D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security         -fstack-clash-protection -fcf-protection -Wp,-D_GLIBCXX_ASSERTIONS -fpermissive -DSUPPORT_TOPO_STREAM_OPERATORS -Wno-deprecated-declarations -fopenmp -O3 -DNDEBUG -fPIC -fPIC -std=c++14 -MD -MT plugins/core/Standard/qAnimation/extern/QTFFmpegWrapper/CMakeFiles/QTFFMPEG_LIB.dir/QVideoEncoder.cpp.o -MF plugins/core/Standard/qAnimation/extern/QTFFmpegWrapper/CMakeFiles/QTFFMPEG_LIB.dir/QVideoEncoder.cpp.o.d -o plugins/core/Standard/qAnimation/extern/QTFFmpegWrapper/CMakeFiles/QTFFMPEG_LIB.dir/QVideoEncoder.cpp.o -c /home/toni/.cache/yay/cloudcompare-git/src/cloudcompare/plugins/core/Standard/qAnimation/extern/QTFFmpegWrapper/QVideoEncoder.cpp
/home/toni/.cache/yay/cloudcompare-git/src/cloudcompare/plugins/core/Standard/qAnimation/extern/QTFFmpegWrapper/QVideoEncoder.cpp: In static member function ‘static bool QVideoEncoder::GetSupportedOutputFormats(std::vector<QVideoEncoder::OutputFormat>&, bool)’:
/home/toni/.cache/yay/cloudcompare-git/src/cloudcompare/plugins/core/Standard/qAnimation/extern/QTFFmpegWrapper/QVideoEncoder.cpp:112:50: error: ‘av_oformat_next’ was not declared in this scope; did you mean ‘av_opt_next’?
  112 |                         AVOutputFormat* format = av_oformat_next(prev);
      |                                                  ^~~~~~~~~~~~~~~
      |                                                  av_opt_next
/home/toni/.cache/yay/cloudcompare-git/src/cloudcompare/plugins/core/Standard/qAnimation/extern/QTFFmpegWrapper/QVideoEncoder.cpp: In member function ‘bool QVideoEncoder::open(QString, QStringList&)’:
/home/toni/.cache/yay/cloudcompare-git/src/cloudcompare/plugins/core/Standard/qAnimation/extern/QTFFmpegWrapper/QVideoEncoder.cpp:162:47: warning: invalid conversion from ‘const AVOutputFormat*’ to ‘AVOutputFormat*’ [-fpermissive]
  162 |                 outputFormat = av_guess_format(qPrintable(formatShortName), NULL, NULL);
      |                                ~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                               |
      |                                               const AVOutputFormat*
/home/toni/.cache/yay/cloudcompare-git/src/cloudcompare/plugins/core/Standard/qAnimation/extern/QTFFmpegWrapper/QVideoEncoder.cpp:195:47: warning: invalid conversion from ‘const AVCodec*’ to ‘AVCodec*’ [-fpermissive]
  195 |         AVCodec *pCodec = avcodec_find_encoder(codec_id);
      |                           ~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~
      |                                               |
      |                                               const AVCodec*
/home/toni/.cache/yay/cloudcompare-git/src/cloudcompare/plugins/core/Standard/qAnimation/extern/QTFFmpegWrapper/QVideoEncoder.cpp:245:28: error: ‘AVStream’ {aka ‘struct AVStream’} has no member named ‘codec’
  245 |         m_ff->videoStream->codec = m_ff->codecContext;
      |                            ^~~~~
/home/toni/.cache/yay/cloudcompare-git/src/cloudcompare/plugins/core/Standard/qAnimation/extern/QTFFmpegWrapper/QVideoEncoder.cpp: In member function ‘virtual bool QVideoEncoder::close()’:
/home/toni/.cache/yay/cloudcompare-git/src/cloudcompare/plugins/core/Standard/qAnimation/extern/QTFFmpegWrapper/QVideoEncoder.cpp:320:27: error: ‘avcodec_encode_video2’ was not declared in this scope; did you mean ‘avcodec_encode_subtitle’?
  320 |                 int ret = avcodec_encode_video2(m_ff->codecContext, &pkt, 0, &got_packet);
      |                           ^~~~~~~~~~~~~~~~~~~~~
      |                           avcodec_encode_subtitle
/home/toni/.cache/yay/cloudcompare-git/src/cloudcompare/plugins/core/Standard/qAnimation/extern/QTFFmpegWrapper/QVideoEncoder.cpp:334:42: error: ‘AVStream’ {aka ‘struct AVStream’} has no member named ‘codec’
  334 |         avcodec_close(m_ff->videoStream->codec);
      |                                          ^~~~~
/home/toni/.cache/yay/cloudcompare-git/src/cloudcompare/plugins/core/Standard/qAnimation/extern/QTFFmpegWrapper/QVideoEncoder.cpp:340:60: error: ‘AVStream’ {aka ‘struct AVStream’} has no member named ‘codec’
  340 |                 av_freep(&m_ff->formatContext->streams[i]->codec);
      |                                                            ^~~~~
/home/toni/.cache/yay/cloudcompare-git/src/cloudcompare/plugins/core/Standard/qAnimation/extern/QTFFmpegWrapper/QVideoEncoder.cpp: In member function ‘virtual bool QVideoEncoder::encodeImage(const QImage&, int, QString*)’:
/home/toni/.cache/yay/cloudcompare-git/src/cloudcompare/plugins/core/Standard/qAnimation/extern/QTFFmpegWrapper/QVideoEncoder.cpp:381:27: error: ‘avcodec_encode_video2’ was not declared in this scope; did you mean ‘avcodec_encode_subtitle’?
  381 |                 int ret = avcodec_encode_video2(m_ff->codecContext, &pkt, m_ff->frame, &got_packet);
      |                           ^~~~~~~~~~~~~~~~~~~~~
      |                           avcodec_encode_subtitle
[11/543] Building CXX object plugins/core/IO/qE57IO/extern/libE57Format/CMakeFiles/E57Format.dir/src/E57XmlParser.cpp.o

```

--- a/.SRCINFO
+++ b/.SRCINFO
@@ -1,6 +1,6 @@
 pkgbase = cloudcompare-git
 	pkgdesc = A 3D point cloud (and triangular mesh) processing software
-	pkgver = 2.10.2.r1025.g08b7ae9a
+	pkgver = 2.10.2.r1076.g541800c9
 	pkgrel = 1
 	url = http://www.danielgm.net/cc/
 	arch = i686
@@ -17,7 +17,7 @@ pkgbase = cloudcompare-git
 	makedepends = python
 	depends = cgal
 	depends = fbx-sdk
-	depends = ffmpeg
+	depends = ffmpeg4.4
 	depends = glew
 	depends = glu
 	depends = mesa

--- a/PKGBUILD
+++ b/PKGBUILD
@@ -8,13 +8,13 @@ name=cloudcompare
 #_fragment="#branch="
 options=('!strip') # strip would also remove plugins, for some reason
 pkgname=${name}-git
-pkgver=2.10.2.r1025.g08b7ae9a
+pkgver=2.10.2.r1076.g541800c9
 pkgrel=1
 pkgdesc="A 3D point cloud (and triangular mesh) processing software"
 arch=('i686' 'x86_64')
 url="http://www.danielgm.net/cc/"
 license=('GPL2')
-depends=('cgal' 'fbx-sdk' 'ffmpeg' 'glew' 'glu' 'mesa' 'mpir' 'pcl' 'pdal' 'qt5-base' 'qt5-tools' 'qt5-svg' 'qt5-websockets' 'shapelib' 'tbb' 'opencascade' 'opencv' 'gdal')
+depends=('cgal' 'fbx-sdk' 'ffmpeg4.4' 'glew' 'glu' 'mesa' 'mpir' 'pcl' 'pdal' 'qt5-base' 'qt5-tools' 'qt5-svg' 'qt5-websockets' 'shapelib' 'tbb' 'opencascade' 'opencv' 'gdal')
 #fix pcl:>vtk:>vtkm:>openmp missing deps
 depends+=(openmp)
 makedepends=('clang' 'cmake' 'doxygen' 'git' 'laz-perf' 'libharu' 'ninja' 'proj' 'python')
@@ -85,6 +85,8 @@ build() {
         -DPLUGIN_STANDARD_QCORK:BOOL=ON # require mpir, cork (cork-git is not enough)
         -DMPIR_INCLUDE_DIR:PATH=/usr/include # required by qcork plugin
         -DCORK_INCLUDE_DIR:PATH="${srcdir}/${name}-cork/src" # required by qcork plugin
+        -DFFMPEG_INCLUDE_DIR:PATH=/usr/include/ffmpeg4.4 # required by qAnimation plugin
+        -DFFMPEG_LIBRARY_DIR:PATH=/usr/lib/ffmpeg4.4 # required by qAnimation plugin
         -DCORK_RELEASE_LIBRARY_FILE:FILEPATH="${srcdir}/${name}-cork/lib/libcork.a" # required by qcork plugin
         -DMPIR_RELEASE_LIBRARY_FILE:FILEPATH=/usr/lib/libmpir.so # require by qcork plugin
         -DPLUGIN_STANDARD_QCSF:BOOL=ON

